kolibri/plugins/user/assets/src/views/sign-up-page/index.vue

```diff
@@ -373,4 +373,7 @@
     margin: 36px
     margin-top: 96px
 
+  button[type='submit']
```

Comment:
Recommend in general adding style-specific class names rather than using html-based selectors. This supports safer template refactoring.

For example, if someone comes along in the future and edits the template to e.g. remove `submit` or (gasp) not change `button` to `div`, they might not realize that will remove some style targeted at it. However if it has a class associated with it, they know to check the additional styles below.